index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>week-planner</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="title">
+    <h1>Weekly Planner</h1>
+  </div>
+  <div class="week">
+    <h2>Sunday</h2>
+    <h2>Monday</h2>
+    <h2>Tuesday</h2>
+    <h2>Wednesday</h2>
+    <h2>Thursday</h2>
+    <h2>Friday</h2>
+    <h2>Saturday</h2>
+  </div>
+  <div class="button">
```

Comment:
this class name is also a bit misleading it would be better as `button-container`